kong/plugins/aws-lambda/api-gateway/aws/AWSBasicCredentials.lua

```diff
@@ -0,0 +1,42 @@
+--[[
+  Copyright 2016 Adobe Systems Incorporated. All rights reserved.
```

Comment:
Agreed, @Tieske . [https://github.com/Mashape/kong/blob/bbaea4ca0a62b2050b0457af8787b98b48349a76/kong/vendor/classic.lua#L4](classic.lua, Copyright (c) 2014, rxi) is already exhibiting this. Also, all dependencies in the rockspec. Except the api-gateway files are Apache 2.0 license whereas the classic.lua is MIT.

Any feedback/suggestions about whether/how better to include this? Or is this a no-go? I understand and called out in the README.md that it seemed odd. This reminds me that this is not a purely open source project, much like adobe's project. I leave it to someone smarter/more entangled than myself to make that call. Rewriting that portion *is* something I'm interested in, but would take a while for obvious reasons, and would probably need to be dependent on other projects at some level for certain things like the crypto.